src/AWS.Deploy.ServerMode.Client/AWS.Deploy.ServerMode.Client.csproj

```diff
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.0</TargetFrameworks>
```

Comment:
For consistency, can you switch to netcoreapp3.1 ?

Reply:
This library will eventually need to run in Visual Studio 2019+. That means we have to keep its target to .NET Standard 2.0.